BlazarService/src/main/java/com/hubspot/blazar/guice/BlazarServiceModule.java

```diff
@@ -91,34 +88,33 @@ public void configure(Binder binder) {
     binder.bind(ObjectMapper.class).toInstance(getEnvironment().getObjectMapper());
     Multibinder.newSetBinder(binder, ContainerRequestFilter.class).addBinding().to(GitHubNamingFilter.class).in(Scopes.SINGLETON);
 
+    Multibinder<RepositoryBuildVisitor> repositoryBuildVisitorMultibinder = Multibinder.newSetBinder(binder, RepositoryBuildVisitor.class);
```

Comment:
The core idea of the Multibinder is that each module that has to contribute a binding is creating the multibinder and adds its binding. You don't need to pass over the same instance between the modules that add their bindings. The multibinder is registered on a class (or interface) so Guice understands and adds the binding under the same set for a class even if in each module you do Multibinder.newSetBinder() 
From https://github.com/google/guice/wiki/Multibindings
```Note: The method Multibinder.newSetBinder(binder, type) can be confusing. This operation creates a new binder, but doesn't override any existing bindings. A binder created this way contributes to the existing Set of implementations for that type. It would create a new set only if one is not already bound.```